samples/charts/financial-chart/tooltip-types/Pages/FinancialChartTooltipTypes.razor

```diff
@@ -0,0 +1,47 @@
+﻿@page "/FinancialChartTooltipTypes"
+@* this namespace is required for Infragistics controls *@
+@using Infragistics.Blazor.Controls
+@using Samples.Services
+@inject IInfragisticsBlazor InfragisticsBlazor;
+@inject IJSRuntime JSRuntime;
+
+<div class="igContainer">
+    <div class="igOptions">
+        <label class="igOptions-label">Tooltip Type: </label>
+        <select @bind="@TooltipType">
+            <option>@ToolTipType.Default</option>
+            <option>@ToolTipType.Item</option>
```

Comment:
swap position of above 2 lines because this sample is using ToolTipType.Item as default tooltip type

Reply:
pending, moved category type too so default is next to none. 

Item
Category
Default
None